terraform/deployments/cluster-services/eks_access.tf

```diff
@@ -87,6 +98,23 @@ resource "aws_eks_access_policy_association" "licensing" {
   ]
 }
 
+resource "aws_eks_access_policy_association" "readonly" {
+  for_each = data.aws_iam_roles.readonly.arns
+
+  cluster_name  = local.cluster_name
+  policy_arn    = "arn:aws:eks::aws:cluster-access-policy/AmazonEKSViewPolicy"
+  principal_arn = each.value
+
+  access_scope {
+    type       = "namespace"
+    namespaces = local.developer_namespaces
+  }
+
+  depends_on = [
+    aws_eks_access_entry.developer
```

Comment:
```suggestion
    aws_eks_access_entry.readonly
```

Reply:
That'll teach me to be a copyin' and a pastin'